.travis.yml

```diff
@@ -14,19 +13,17 @@ before_script:
   - ./cc-test-reporter before-build
   - 'bundle exec rake db:migrate SINATRA_ENV=test'
 script:
-  - 'bundle exec rake $CHECK'
+  - 'bundle exec rspec'
```

Comment:
does this make sense? Later on we set `env: CHECK=test`

Reply:
We currently only have one check that we run on different ruby versions. I can remove the $CHECK

Reply:
I think that makes sense. the travis config currently references the rake tasks rspec, test and test_and_report_coverage

Reply:
Ahh, `test` runs both `rubocop` and `rspec`. Probably should run rubocop as a standalone test